articles/user-guide/language-ref/attributes/entrypoint-ref.md

```diff
@@ -0,0 +1,16 @@
+---
+title: '@EntryPoint'
+description: Keyword reference '@EntryPoint'
+author: gillenhaalb
+ms.author: a-gibec
+ms.date: 05/09/2020
+ms.topic: article
+uid: microsoft.quantum.lang-ref.entrypoint
+---
+
+# `@EntryPoint`
```

Comment:
This and `@Test` are both attributes that are already documented in the API reference; can we link to those from the TOC rather than duplicating that content?

Reply:
@cgranade That's a great point, thanks. Just to clarify, it sounds like you mean keeping the links in the TOC dropdown (and the keyword list page), but simply linking direct to the API page?

Reply:
I think that should work, yeah. If I recall, you can make `href` properties in `toc.yml` point to xrefs instead of Markdown file paths; that would allow for reusing the automatically generated references for Test and EntryPoint.